src/tidal_mcp/auth.py

```
@@ -0,0 +1,558 @@
+            
+        except Exception as e:
+            logger.error(f"OAuth2 flow failed: {e}")
+            return False
+    
+    async def _capture_auth_code(self) -> Optional[str]:
+        """Start local server to capture OAuth2 callback."""
+        from aiohttp import web
+        import asyncio
+        
+        auth_code = None
+        server = None
```

Ruff (0.12.2)

Local variable `server` is assigned to but never used

Remove assignment to unused variable `server`

(F841)